### PR TITLE
Single Nucleus Smart-seq2 optimization

### PR DESCRIPTION
**Pull request by khajoue2**
Removed single sample pipeline
Changed wdl tasks to allow multiple samples with each VM run

### CODE REVIEW

**Review by ekiernan — COMMENTED** [review]
We need to merge the single sample pipeline and multi-sample pipeline documentation before merging this in to avoid confusion from users.

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl [hunk 1]
```diff
@@ -100,11 +135,13 @@ workflow MultiSampleSmartSeq2SingleNucleus {
   }
 
 
+
+
   ### Pipeline output ###
   output {
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl:141]: Can we add the pipeline version as a string output?

Comment (jessicaway) [reply pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl:141]: This seems like a good idea

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 1.1.0
+2021-09-16 (Date of Last Commit)
+
+* Removed the Smart-seq2 Single Nucleus workflow (SmartSeq2SingleNucleus.wdl) and changed the workflow tasks to run multiple samples in the same VM
```
Comment (jessicaway) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:4]: maybe comment that this is expected to make the pipeline faster and cheaper

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 5.1.1
+2021-09-13 (Date of Last Commit)
+
+* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2
```
Comment (jessicaway) [inline pipelines/skylab/optimus/Optimus.changelog.md:4]:
```suggestion
* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2. These changes do not affect Optimus
```

File: pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 2.2.6
+2021-09-13 (Date of Last Commit)
+
+* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2
```
Comment (jessicaway) [inline pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md:4]:
```suggestion
* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2. These changes do not affect MultiSampleSmartSeq2
```

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl [hunk 5]
```diff
@@ -100,11 +135,13 @@ workflow MultiSampleSmartSeq2SingleNucleus {
   }
 
 
+
```
Comment (jessicaway) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl:138]:
```suggestion
```

File: pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md [hunk 6]
```diff
@@ -1,4 +1,10 @@
+# 5.1.5
+2021-09-13 (Date of Last Commit)
+
+* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2
```
Comment (jessicaway) [inline pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md:4]:
```suggestion
* Updated Picard.wdl and LoomUtils.wdl for Single Nucleus SmartSeq2. These changes do not affect SmartSeq2SingleSample
```

**Review by jessicaway — APPROVED** [review]
Couple minor comments, but looks good

File: tasks/skylab/StarAlign.wdl [hunk 7]
```diff
@@ -131,6 +129,85 @@ task StarAlignFastqPairedEnd {
...
+    Array[File] fastq1_input_files
+    Array[File] fastq2_input_files
+    Array[String] input_ids
+    File tar_star_reference
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.9a"
+    Int machine_mem_mb = ceil((size(tar_star_reference, "Gi")) + 6) * 1100
+    Int cpu = 16
+    # multiply input size by 2.2 to account for output bam file + 20% overhead, add size of reference.
```
Comment (nikellepetrillo) [inline tasks/skylab/StarAlign.wdl:143]:
```suggestion
    # multiply input size by 2.0 to account for output bam file + 20% overhead, add size of reference.
```

**Review by nikellepetrillo — APPROVED** [review]
looks great!

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md (hunk 2 above)
Comment (khajoue2) [reply pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md]: done

**Pull request merged**